### ISSUE TITLE
Broken for A-Frame 0.8.0

### ISSUE THREAD

**Issue by erickzhao** [takahirox/aframe-outline#6]
Hey there, it seems like this component doesn't work with versions `0.8.0` and up of A-Frame. I copied over your demo and bumped the A-Frame version number up to the latest one and no outlines seem to be appearing regardless of my settings.

[Link to demo.](https://jsfiddle.net/d2uL0w8p/)

**Comment by takahirox** [comment]
Thanks for reporting. I'll look into.

**Comment by MannyMeadows** [comment]
Looks like we're still out of action for `0.9.0`. Not sure where it's falling over though.

**Comment by asandersSAP** [comment]
Yes, still broken in 0.9.0. It's the "effect" field of the sceneEl which seems to be deprecated since 0.8.0

**Comment by jrDev1** [comment]
Hello,

Does this still work?

Thanks,
jrDev

**Comment by Mocio89** [comment]
Hello,

Any updates?

Thanks
